### PR TITLE
MAIN-6066 add missing test

### PR DESCRIPTION
**Pull request by sebastianmarzjan**
Will fail if the incident reoccurs. Unfortunately, I don't have a better idea at the moment to handle this.

### CODE REVIEW

**Comment by nikodamn** [comment]
LGTM

File: includes/wikia/parser/templatetypes/tests/DataTablesTest.php [hunk 1]
```diff
@@ -42,6 +42,16 @@ public function testTablesMarking( $html, $expected ) {
 		$this->assertEquals( $expected, $result );
 	}
 
+	/**
```
Comment (RafLeszczynski) [inline includes/wikia/parser/templatetypes/tests/DataTablesTest.php:45]: please consider adding a comment why this test is needed

**Comment by RafLeszczynski** [comment]
one small remark except that LGTM

File: includes/wikia/parser/templatetypes/tests/DataTablesTest.php (hunk 1 above)
Comment (sebastianmarzjan) [inline includes/wikia/parser/templatetypes/tests/DataTablesTest.php:45]: Done in 376f265

**Comment by RafLeszczynski** [comment]
LGTM

**Pull request merged**